src/Resources/views/Common/_box.html.twig

```diff
@@ -21,7 +21,7 @@
         <a href="#" class="header sylius-product-name">{{ result.attribute('name').value | join }}</a>
 
         {% if result.price is not empty %}
-            <div class="sylius-product-price">{{ money.format(result.priceByChannelAndCurrency(channel.code, currencyCode).value, currencyCode) }}</div>
+            <div class="sylius-product-price">{{ money.convertAndFormat(result.priceByChannelAndCurrency(channel.code, currencyCode).value, currencyCode) }}</div>
```

Comment:
Could you update the call to fit the method signature please?

https://github.com/Sylius/Sylius/blob/981021539e4e66a97f8b1d1d8ebed8dead1b3d96/src/Sylius/Bundle/ShopBundle/Resources/views/Common/Macro/money.html.twig#L5-L9

Reply:
Resolved.

Reply:
Sorry but no, it's not. Did you forget to push?